src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Ball extends ColorSopplier {
```

Comment:
Ball should not extend CollorSupplier, it should be used as a separate object instance just to get a random color and pass it to the constructor of the Ball